.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: scala
 scala:
-  - 2.9.2
+  - 2.12.1
```

Comment:
bah, still need to add a java target i guess:

https://travis-ci.org/SimpleFinance/jdub/builds/188967494#L165